frontend/vue.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    devServer: {
+        proxy: "http://localhost:9090"
```

Comment:
Tutaj też trzeba ustawić na zzpj-backend i tylko jak pracujemy lokalnie to sobie to podmieniać.

Reply:
W ogóle url'e zwykle przechowuje się w pliku .env zamiast w pliku js

Reply:
Proxy działa tylko na wersji develop. Gdy uruchamiamy front jako docker to się odpala wersja production i tutaj nawet nie wchodzi. Narazie walczymy z Wróblem jak to naprawić.